src/lib.rs

```diff
@@ -10,7 +10,10 @@ use hal_core::{boot::BootInfo, mem};
 
 use alloc::vec::Vec;
 
-pub fn kernel_main(bootinfo: &impl BootInfo) -> ! {
+mod wasm;
+
+const HELLOWORLD_WASM: &[u8] = include_bytes!(concat!(env!("OUT_DIR"), "/helloworld.wasm"));
+
```

Comment:
I'm pretty sure this change won't build, unless I'm missing something else in this interdiff.

Reply:
uwu, i did a widdle fucky wucky

Reply:
should be fixed in 9b85256c2deb0b1a6cd6bfd18233f57306527542